spec/codebreaker/guess_checker_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+RSpec.describe Codebreaker::GuessChecker do
+  test_data = [
+    [%w[5 6 4 3], %w[6 5 4 3], { index: 2, include: 2 }],
```

Comment:
use hashes pls do identify secret code and guess

Reply:
fixed